src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final int PARAMETER_INDEX_1 = 1;
+    private static final int PARAMETER_INDEX_2 = 2;
+    private static final int PARAMETER_INDEX_3 = 3;
+    private static final String COLUMN_LABEL_ID = "id";
+    private static final String COLUMN_LABEL_NAME = "name";
+    private static final String COLUMN_LABEL_COUNTRY = "country";
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
```

Comment:
I would name it insertManufacturerQuery . It is better to use query word instead request, request usually means http request. Also let's fix your query syntax (fix the same issue in all places):
```suggestion
        String insertManufacturerRequest = "INSERT INTO manufacturers (name, country) VALUES(?, ?)";
```